unlock-app/jest.setup.ts

```diff
@@ -1,7 +1,6 @@
 /* eslint-disable import/no-extraneous-dependencies */
 
-import 'jest-dom/extend-expect'
-import 'react-testing-library/cleanup-after-each'
```

Comment:
This import is deprecated under the new library. Also changed to TS here so that we could get type inference for jest-dom